src/python/gudhi/representations/archipelago.py

```diff
@@ -0,0 +1,119 @@
+# This file is part of the Gudhi Library - https://gudhi.inria.fr/ - which is released under MIT.
+# See file LICENSE or go to https://gudhi.inria.fr/licensing/ for full license details.
+# Author(s):       Martin Royer
+
+import copy
+
+import numpy as np
+
+from sklearn.base import BaseEstimator, TransformerMixin
+
+from gudhi.representations.vector_methods import Atol, TopologicalVector, BettiCurve
+
+
+class Archipelago(BaseEstimator, TransformerMixin):
+    """
+    Transformer that dictionary-wraps persistence diagram vectorizers, i.e. objects from gudhi.representations.vector_methods.
```

Comment:
(this may be more natural with the new format)
Maybe we should say that this class is helpful to avoid going through DimensionSelector+FeatureUnion or similar. It could even be nice to have an example somewhere comparing the 2 (we can add it later though).